examples/testapp/src/context/CBWSDKReactContextProvider.tsx

```diff
@@ -133,17 +167,32 @@ export function CBWSDKReactContextProvider({ children }: CBWSDKProviderProps) {
       setSDKVersion,
       scwUrl,
       setScwUrlAndSave,
+      config,
+      setConfig,
     }),
-    [sdk, provider, option, setPreference, version, setSDKVersion, scwUrl, setScwUrlAndSave]
+    [
+      sdk,
+      provider,
+      option,
+      setPreference,
+      version,
+      setSDKVersion,
+      scwUrl,
+      setScwUrlAndSave,
```

Comment:
Should we add `config` to deps?

Reply:
yes. we need to linting fixed in the SDK. I have a separate task to look into it